docs/recipes/RemoveMobxStateTree.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Remove Mobx-State-Tree
+description: How to remove Mobx-State-Tree from an Ignite project
+tags:
+  - MobX
+  - State Management
+last_update:
+  author: Justin Poliachik
+publish_date: 2024-02-05
+---
+
+# Remove Mobx-State-Tree
+
+By default, Ignite uses [Mobx-State-Tree](https://mobx-state-tree.js.org/) as the default state management solution. While we love [Mobx-State-Tree at Infinite Red](https://docs.infinite.red/ignite-cli/concept/MobX-State-Tree/), we understand the landscape is rich with great alternatives that you may want to use instead.
+
+This guide will show you how to remove Mobx-State-Tree from an Ignite-generated project and get to a "blank slate" with no state management at all.
+
+## Steps
+
+1. Let's start by removing all Mobx-related dependencies from `package.json`, then run `yarn` or `npm i`
```

Comment:
Thoughts on just providing the command here rather than making the user search through the dependency list? Easier copy/paste grab for the user

For example: 
`yarn remove mobx mobx-react-lite mobx-state-tree reactotron-mst`

Reply:
agreed, I like this better!